php/PHPCD.php

```diff
@@ -319,34 +334,42 @@ private function getMethodInfo($method, $pattern = null)
             return $param->getName();
         }, $method->getParameters());
 
-        $modifier = $this->getModifier($method);
+        $modifier = $this->getModifiers($method);
 
         return [
             'word' => $name,
-            'abbr' => "$modifier $name (" . join(', ', $params) . ')',
+            'abbr' => sprintf("%s\t\t%s (%s)", $modifier, $name, join(', ', $params)),
```

Comment:
`%s\t\t%s (%s)" will be not suitable for some case like
```
final static private function hehe() {}
```

So, what about to use `%4s %s (%s)`?